### PR TITLE
Add example for Bucket.Delete() in README.md

### PR DESCRIPTION
**Pull request by cenkalti**
Resurrect #338

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -336,7 +336,17 @@ exists then it will return its byte slice value. If it doesn't exist then it
 will return `nil`. It's important to note that you can have a zero-length value
 set to a key which is different than the key not existing.
 
-Use the `Bucket.Delete()` function to delete a key from the bucket.
+Use the `Bucket.Delete()` function to delete a key from the bucket:
+
+```go
+db.Update(func (tx *bolt.Tx) error {
+    b := tx.Bucket([]byte("MyBucket"))
+    err := b.Delete([]byte("answer"))
```
Comment (ahrtr) [inline README.md:344]: When the bucket "MyBucket" doesn't exist, then the returned `b` is nil; accordingly `b.Delete` will panic due to nil pointer error. I am thinking we should add a check something like `if b != nil`. 

All examples in README do not have such check, so we can address this in a separate PR to update all examples. WDYT?

**Review by ahrtr — APPROVED** [review]
LGTM with a minor comment. Please feel free to address it in a separate PR.

Thank you @cenkalti

File: README.md (hunk 1 above)
Comment (cenkalti) [reply README.md:344]: I think nil check can be omitted in examples for simplicity.

Comment (ahrtr) [reply README.md:344]: OK, makes sense.

**Pull request merged**